### PR TITLE
Feature/freemind 1.0.1

### PR DESCRIPTION
**Pull request by compleatguru**
Updated with README.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,7 +12,7 @@ for display by Freemind. It was written out of annoyance with the Freemind user
 interface, and the lack of 'merging' capabilities when collaborating with other
 people.
 
-Copyright  2006–2008  Wouter Bolsterlee <uws@xs4all.nl>
+Copyright  2006–2017  Wouter Bolsterlee <uws@xs4all.nl>
```
Comment (wbolster) [inline README.rst:15]: nice touch, thank you

File: text-to-freemind [hunk 2]
```diff
@@ -126,7 +126,7 @@ def convert_lines_into_mm(lines, out_fp):
             n = et.SubElement(curpath[-1], 'node')
             curpath.append(n)
             set_node_properties(n, line)
-            
+
```
Comment (wbolster) [inline text-to-freemind:129]: :+1:

**Review by wbolster — APPROVED** [review]

**Comment by wbolster** [comment]
merged, thanks @compleatguru!

**Comment by wbolster** [comment]
(for reference: this pr is an improved version of #1)

**Comment by compleatguru** [comment]
Thank you!

I am glad to contribute to your nice converter!

**Comment by wbolster** [comment]
to be honest i forgot i wrote this so i am happily surprised someone finds it useful!

**Pull request merged**